libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -197,6 +229,10 @@ void AP_Baro::update_calibration()
             _EAS2TAS = 0;
         }
     }
+
+    // update and save base pressure (persistent between boots) with primary baro ground calibration (not persistent)
+    _base_pressure.set_and_save(get_ground_pressure());
+    _base_pressure.notify();
```

Comment:
I think we can get rid of both BASE_PRESSURE and BASE_RESET. But there will need to be some tweaks in Baro to accomplish this. One problem is that there are not separate parameters for the separate sensors (like compass, as you mentioned). Another problem is that calibrate is expected to be called at least once at boot before updating and reading the sensor.